### DISCUSSION TITLE
Podcast Support??

### DISCUSSION THREAD

**Discussion by AndreDio** [jagrosh/MusicBot#1035] [Questions]
Not sure if I'm just doing it wrong, but podcasts won't play. Tried multiple platforms, all say unknown format. Are podcasts not supported? If so can we expect support to be added in the future?

**Comment by AdrX003** [comment]
On my case i would just download the episodes and play them locally just as songs. I guess most of the sites that offers podcasts are made with some different parts as the already working sites described in the wiki that the bot can play from. To add them for streaming maybe make a list of the most used and the ones you use and ask for support of these?
